public/styles/styles.css

```diff
@@ -146,7 +146,99 @@ body {
     text-align: center;
     text-shadow: 2px 2px 30px var(--red);
 }
+/* Footer begins here */
```

Comment:
@ZM202 We need to check out the general styling of the footer. Seems that a few elements are jumping out of the container. I think you had a quick fix before?

Reply:
@kststudio I updated overall layout of the footer. Pls let me know if anything seems out of elements. Thanks.